src/karts/kart.cpp

```diff
@@ -1375,6 +1376,7 @@ void Kart::update(float dt)
             fabs(getSpeed()) < 3.0f)
         {
             new RescueAnimation(this, /*is_auto_rescue*/true);
+            m_last_factor_engine_sound=0.0f;
```

Comment:
This may seem minor but our code convention requires spaces around the "=", could you please just add it to follow our conventions?